map/src/data/markers.ts

```diff
@@ -1,29 +1,46 @@
 import { Service } from './index';
 
+/**
+ * Contact details capture various methods to contact an organization.
+ */
 export interface ContactDetails {
   facebookGroup?: string;
-  web?: { [id: string]: string };
+  web?: { [id: string]: string }; // List of URLs
   phone?: string[];
   email?: string[];
 }
 
+/**
+ * Locations can be saved so they can be re-used for other markers.
+ *
+ * See the LOCATIONS array.
+ */
 export interface Location {
-  description: string;
+  description: string; // Human readable name for the location -- displayed on the web.
   lat: number;
   lng: number;
-  serviceRadius: number;
+  serviceRadius: number; // Measured in Meters (per Google Maps standard)
 }
 
+/**
+ * A marker that will be rendered on the map. A short title and description is also visible to users.
+ *
+ * It contains an array of services
+ */
 export interface MarkerInfo {
+  // name of the organization or community effort
```

Comment:
you probably want to use TSDoc syntax `/**` so that these hints are actually rendered for references to these types, but this is good enough for now.

Reply:
/** has to be done at the top of the file ?